sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/converters/CommunicationIdentifierConverter.java

```diff
@@ -77,13 +79,13 @@ private static void assertSingleType(CommunicationIdentifierModel identifier) {
     }
 
     /**
-     * Serialize CommunicationIdentifier into CommunicationIdentifierModel
-     * @param identifier CommunicationIdentifier object to be serialized
+     * Convert CommunicationIdentifier into CommunicationIdentifierModel
```

Comment:
nice !. Can you give as a hand making sure the other sdks (chat) use the same name please ...